### PR TITLE
first attempt at nimish problem with subarrays and master sky

### PR DESCRIPTION
**Pull request by Russell-Ryan**
This is a first attempt at catching the error of master sky subtraction with a subarray-ed image.

### CODE REVIEW

File: slitlessutils/core/modules/extract/single/single.py [hunk 1]
```diff
@@ -115,14 +115,13 @@ def __init__(self, extorders, mskorders='all', savecont=False, root=None,
...
-                LOGGER.warning('Cannot make CSVPATH, so no CSV files will be written')
-                self.writecsv = False
-                self.csvpath = None
-
-        print(self.outpath, self.writecsv)
+            if not os.path.exists(self.csvpath):
+                try:
+                    os.mkdir(self.csvpath)
+                except FileNotFoundError:
+                    LOGGER.warning('Cannot make CSVPATH, so no CSV files will be written')
```
Comment (rosteen) [inline slitlessutils/core/modules/extract/single/single.py:122]:
```suggestion
                    LOGGER.warning(f'Cannot make CSVPATH {self.csvpath},'
                                   ' so no CSV files will be written')
```

May as well give the path that's failing here.

**Review by rosteen — COMMENTED** [review]
Made one small suggestion for more info in a Logger message. Also, given that you're removing a lot of commented-out old code here, should the new commented-out blocks be removed as well? I see new commented blocks in `affinetweak.py` and 'logger.py'.

File: slitlessutils/core/modules/extract/single/single.py (hunk 1 above)
Comment (Russell-Ryan) [reply slitlessutils/core/modules/extract/single/single.py:122]: totally.  Good idea.

**Review by rosteen — APPROVED** [review]
@nphathi gave the thumbs-up offline, approved.

**Pull request merged**